SONIC-DEV/sonic-buildimage
# [platform/cel]: Update watchdog service

**- What I did**
-  Update watchdog service

**- How I did it**
-  Add debug message
-  Add retry mechanism for heartbeat request

**- How to verify it**
- Run command ` cat /var/log/bmcwdt.log ` on SONIC to check watchdog service log
- Run command `systemctl status bmc_wdt.service` on SONIC to check service status
- Run command `tail /var/log/messages` on BMC to view watchdog enable/disable log
